api/tools/src/main/java/org/pmiops/workbench/tools/SetAccessModuleTimestamps.java

```diff
@@ -60,42 +66,54 @@ void applyTimestampUpdateToUser(
       AccessModuleName moduleName,
       @Nullable Timestamp timestamp,
       boolean isBypass) {
-    DbUser dbUser = userDao.findUserByUsername(username);
-    if (dbUser == null) {
-      throw new IllegalArgumentException(String.format("User %s not found!", username));
-    }
-
-    if (moduleName == AccessModuleName.PROFILE_CONFIRMATION && !isBypass) {
-      final String field = isBypass ? "bypass" : "completion";
-      final String time = Optional.ofNullable(timestamp).map(Timestamp::toString).orElse("NULL");
-      LOG.info(
-          String.format(
-              "Updating %s %s time for user %s to %s", moduleName, field, username, time));
-
+    accessModuleService.updateCompletionTime(dbUser, moduleName, timestamp);
+    accessModuleService.updateBypassTime(
```

Comment:
We don't want to update the bypass time for PROFILE_CONFIRMATION.  It's not bypassable.

Reply:
if not bypassable, accessModuleService will reject it: https://github.com/all-of-us/workbench/blob/a2de6496019564bd98fca3108494d88d6576a47a/api/src/main/java/org/pmiops/workbench/access/AccessModuleServiceImpl.java#L69

Reply:
That's what I mean.  This line will throw an exception.

Reply:
Thanks!. Changed!